src/screens/AddCourseScreen.tsx

```diff
@@ -69,6 +69,23 @@ const AddCourseScreen: React.FC<AddCourseScreenProps> = ({ navigation }) => {
     },
   ];
 
+  const doesCourseOverlap = (newCourse: Course, existingCourses: Course[]): boolean => {
```

Comment:
I would name this `isCourseOverlapped`. I've never seen anyone name boolean function with `does`, though :))

Reply:
Okay, I will change it next time